src/pages/TelaEscolherPet.js

```diff
@@ -0,0 +1,129 @@
+import React, { useContext } from 'react';
+import {
+  Text,
+  View,
+  StyleSheet,
+  SafeAreaView,
+  Image,
+  TouchableOpacity,
+  ScrollView,
+} from 'react-native';
+import { AuthContext } from '../components/Context'
+import PetWrapper from '../components/PetWrapper'
+
+export default function TelaEscolherPet() {
+  const { user } = useContext(AuthContext)
+
+  return (
+    <SafeAreaView style={{ flex: 1 }}>
+      <ScrollView>
+        <View style={styles.blueBar}></View>
+        <View style={styles.navBar}>
+          <TouchableOpacity><Image source={require('./assets/back.png')} /></TouchableOpacity>
+          <TouchableOpacity><Image source={require('./assets/cancel.png')} /></TouchableOpacity>
+        </View>
+        <View style={styles.dogNoExiste}>
+          <Text style={[styles.subtitle, { marginTop: 20 }]}>
+            Escolha o pet que irá para a consulta
+          </Text>
+          <Text style={styles.descriptionColor}>
+            Clique em um desses fofinhos para selecionar
```

Comment:
Aeee, não tinha visto que essa parte do design não estava usando linguagem não assertiva de gênero, já atualizei para isso aqui lá no figma!

```suggestion
            Clique em uma das imagens abaixo para selecionar
```